libr/debug/p/native/xnu/xnu_debug.c

```diff
@@ -198,6 +199,47 @@ int xnu_detach(RDebug *dbg, int pid) {
 #endif
 }
 
+static int task_suspend_count(task_t task) {
+	kern_return_t kr;
+	struct task_basic_info info;
+	mach_msg_type_number_t count = TASK_BASIC_INFO_COUNT;
+	kr = task_info (task, TASK_BASIC_INFO, (task_info_t) &info, &count);
+	if (kr != KERN_SUCCESS) {
+		eprintf ("failed to get task info\n");
+		return -1;
+	}
+	return info.suspend_count;
+}
+
+int xnu_stop(RDebug *dbg, int pid) {
+#if XNU_USE_PTRACE
+	eprintf ("xnu_stop: not implemented\n");
+	return false;
+#else
+	kern_return_t kr;
+	task_t task;
+	int suspend_count;
+
+	task = pid_to_task (pid);
+	if (!task) return false;
```

Comment:
[Style guide](https://github.com/radare/radare2/blame/266686278af37340c0099dc95384a8544dc0413b/CONTRIBUTING.md#L46) seems to permit braceless ifs at the moment (per lines 46, 59–66, and 97–98); might require updating. 😛